dev/com.ibm.ws.persistence-1.0/com.ibm.ws.persistence-1.0.feature

```diff
@@ -4,12 +4,10 @@ visibility=private
 IBM-API-Package: com.ibm.websphere.persistence.mbean; type="ibm-api"
 -features=com.ibm.websphere.appserver.jndi-1.0, \
```

Comment:
this is where we want to be including feature `com.ibm.websphere.appserver.org.eclipse.persistence-2.6` and tolerating `2.7`